hammr/commands/scan/scan.py

```diff
@@ -98,6 +99,19 @@ def do_run(self, args):
             if not doArgs:
                     return 2
 
+            #check if you can add a scan to a scanned instance based on the overlay option
```

Comment:
this part could be extracted to a new function, it will improve readability and be testable !